wai-logger/Network/Wai/Logger/Apache.hs

```diff
@@ -62,6 +75,11 @@ apacheLogStr ipsrc tmstr req status msize =
   <> toLogStr (fromMaybe "" mua)
   <> "\"\n"
   where
+    path =  case pathType of
+      LogOnlyPath ->
+        rawPathInfo req
+      LogPathWithQueryString ->
+        rawPathInfo req <> rawQueryString req
```

Comment:
Yes. I understood your idea.
But I think this is a just bug fix.
So, you can change the current behavior.